pages/wishlist.css

```diff
@@ -0,0 +1,25 @@
+.wishlist-container {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 0rem 2rem;
+  justify-content: space-between;
+}
+
+.product-img-div {
+  position: relative;
+}
+.fa-xmark {
```

Comment:
You can write a more meaningful class for this.

Reply:
yes